roles/test_operator/tasks/main.yml

```diff
@@ -143,15 +143,25 @@
     namespace: "{{ cifmw_test_operator_namespace }}"
     kind: Job
     name: "{{ cifmw_test_operator_tempest_name }}"
-    wait: true
-    wait_timeout: "{{ cifmw_test_operator_timeout }}"
-    wait_condition:
-      type: Complete
-      status: true
+  retries: "{{ (cifmw_test_operator_timeout / 10) | round | int }}"
+  delay: 10
+  until: >
+    tempest.resources[0].status.succeeded | default(0) | int >= 1 or
+    tempest.resources[0].status.failed | default(0) | int >= 1
+  ignore_errors: true
+  register: tempest
   when: not cifmw_test_operator_dry_run | bool
 
-- name: Start test-operator-logs-pod
+- name: Check whether tempest timed out
+  ansible.builtin.set_fact:
+    tempest_timed_out: >-
```

Comment:
is there nothing returned for us here from the tempest.resources ? feels a bit fragile the way you are determining the timeout

Reply:
As we've already discussed there is unfortunately probably no other way how to determine the timeout.